src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx

```diff
@@ -28,21 +28,88 @@ Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capab
+  />
+
+  <TechTile
+    name="PHP agent"
+    icon="logo-php"
+    to="https://one.newrelic.com/nr1-core?state=aa633b41-72d4-009c-3abf-55dcf64894fe"
+  />
+
+  <TechTile
+    name="Python agent"
+    icon="logo-python"
+    to="https://one.newrelic.com/nr1-core?state=20fda75b-58fb-a92a-f9e1-7b052035c6e8"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+</TechTileGrid>        
+
+Once you've installed an agent, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core?filters=(domain%3D'APM'ANDtype%3D'APPLICATION'))** and select your app. If you don't see much data just yet, step away for a while and let the agent gather real-time data as your application runs.
+</Step>
+
+<Step>
 
-1. From the dashboards page, search for the pre-built dashboard you want to customize.
-2. Click the pre-built dashboard to select it.
-3. Select the account in which to add the data and dashboard.
-4. Follow the instructions in the **Setup data source** section. If you've already instrumented the data source, you can skip this step.
-5. Click **View dashboard** to deploy the dashboard and start visualizing your data. Note that you can customize your new pre-built dashboard (see [Manage your dashboard](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/)). You can also change the name and settings, add new content and new widgets, refine the charts, etc.
+### Tell us about your stack [#your-stack]
 
+We will always recommend dashboards we think might be a good fit for your team but if you're looking for a tailored experience that highlights _specific_ dashboards that target your application's unique needs, we recommend telling us a little about how your stack was built. 
 
-## How to view more templates [#more-templates]
+<img
+    title="tell us about your stack"
+    alt="A screenshot depicting how to tell New Relic about how your stack was built"
+    src={dashboardsTellUsAboutYourStack}
+/>
+
+1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Dashboards**.
+2. Select your account in the top right account picker.
+2. Search for the pre-built dashboard subsection.
+3. Click `Tell us about your stack`.
+4. Change your stack configuration by selecting all applicable choices like `apache` or `node.js`.
+2. Save your changes.
+3. New recommendations will now appear on your dashboards homepage.
+</Step>
 
-Browse the entire dashboard catalog to find dashboard templates for all the available integrations. Click the **View All** link next to the popular pre-built dashboards title to see and search for all the available pre-built dashboards. You can use the <Icon name="fe-chevron-left"/> and <Icon name="fe-chevron-right"/> buttons to browse more pre-built dashboards.
+
+<Step>
+### Install pre-built dashboards [#use]
 
 <img
   title="Search for a pre-built dashboard"
```

Comment:
I think that this screenshot should go inside step 4.

Reply:
Sounds good! Moving it